app/models/concerns/Auditable.rb

```diff
@@ -0,0 +1,10 @@
+# Concern Audiable for Models
+Module Auditable.rb
```

Comment:
You can lowercase the word `Module`, and take off the `rb` at the end -- 

`module` is the reserved word, and if it's capitalized, ruby will probably go nuts thinking it's its own object.